jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -124,6 +124,8 @@ def _parse_image(app, file_obj, data_label, ext=None):
     data_iter = get_image_data_iterator(app, file_obj, data_label, ext=ext)
 
     for data, data_label in data_iter:
+        if data.coords is not None:
+            data.coords.bounding_box = None
```

Comment:
@mcara , is there a way to *temporarily* enable extrapolation, like maybe with context manager? Or it has to be all or nothing?

@kecnry , if this is the only way we can solve the display problem, I guess we have no choice but maybe we need to add a warning at https://github.com/spacetelescope/jdaviz/blob/main/docs/imviz/displayimages.rst#cursor-information saying that it might be inaccurate if outside the visible data?